build.py

```diff
@@ -23,8 +23,7 @@
 from vttmisc import tsi1, tsic
 
 VERSION_YEAR_MONTH = 2106
-VERSION_DAY = 03
-
+VERSION_DAY = 0o3
```

Comment:
Whoa. I think we might need `3` instead (0o will blow up or act strange if we ever go above 7 (0o8 is invalid!)). Since it's not a string, the leading 0 was actually doing nothing anyway :smile:

Reply:
Eh heh. Yeah, I was worried that if we push out two releases in a month that the "3" would end up being a 'higher version' than, say "20". Thought this might work, but yeah, nope.

Reply:
(adjusted to `3`)